applications/smart-distancing/Dockerfile-x86

```diff
@@ -1,9 +1,14 @@
-FROM tensorflow/tensorflow:latest-py3
+FROM tensorflow/tensorflow:latest-gpu-py3
 
 VOLUME  /repo
 WORKDIR /repo/applications/smart-distancing
 
-RUN apt-get update && apt-get install -y pkg-config libsm6 libxext6 libxrender-dev
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        libsm6 \
+        libxext6 \
+        libxrender-dev \
+        pkg-config \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
adding `&& apt-get clean` further reduces the image size

Reply:
That used to be true, but newer Debian and Ubuntu images automatically run it at the end of layer.

see note at bottom of run section:
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

I have no idea why they didn't go the extra mile and nuke the apt lists as well. It seems like an oversight.

Reply:
Aha! thanks.